public/content/en/basics/functions.md

```diff
@@ -23,20 +25,20 @@ compatible types.
             return 1.0;
     }
 
-Functions might even be declared inside others functions where they may be
-used locally and aren't visible to the outside world.
-These function can even have access to objects local to
-the parent's scope:
+### Default arguments
 
-    void fun() {
-        int local = 10;
-        int fun_secret() {
-            local++; // that's legal
-        }
+Functions may optionally define default arguments.
+This avoids the tedious work to declare redundant
+overloads. 
+
+    void plot(string msg, string color = "red") {
         ...
+    }
+    plot("D rocks");
+    plot("D rocks", "blue");
 
-Such nested functions are called delegates and will be explained in more depth
-[soon](basics/delegates).
+_After_ arguments with default values following argument
```

Comment:
I get the sentence but it sounds a lottle bit confusing..  "Once a default argument has been specified all followimg arguments must be default arguments too."?